__tests__/app.test.js

```diff
@@ -39,103 +39,175 @@ describe("/api", () => {
-            expect(typeof comment.body).toBe("string");
-            expect(typeof comment.created_at).toBe("string");
-            expect(typeof comment.article_id).toBe("number");
-          })
-          expect(body.comments).toBeSortedBy("created_at", { coerce: true, descending: true });
+describe("/api/articles/:articleId/comments", () => {
+  test("GET:200 sends an array of all comments on the specified article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
         });
-    });
-    test("returns a 404 Not Found when a valid but non-existing id is requested", () => {
-      return request(app)
-        .get("/api/articles/999999/comments")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("No Comments Found");
+        expect(body.comments).toBeSortedBy("created_at", {
+          coerce: true,
+          descending: true,
         });
-    });
-    test("returns a 400 Bad Request when an invalid id is requested", () => {
-      return request(app)
-        .get("/api/articles/banana/comments")
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Bad Request");
+      });
+  });
+  test("GET:200 works when there are no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toEqual([]);
+      });
+  });
+  test("returns a 404 Not Found when a valid but non-existing id is requested", () => {
+    return request(app)
+      .get("/api/articles/999999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Resource Not Found");
+      });
+  });
+  test("returns a 400 Bad Request when an invalid id is requested", () => {
+    return request(app)
+      .get("/api/articles/banana/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("POST /api/articles/:articleId/comments", () => {
+  test("POST returns a 201 and the comment successfully added", () => {
+    const comment = {
+      username: "rogersop",
+      body: "This is a comment",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          body: "This is a comment",
+          author: "rogersop",
+          article_id: 2,
         });
```

Comment:
This is a good test but these 3 properties should not be the only properties on the comment we get back. Your responseExample in endpoints.json seems to have all of the properties we should be expecting.